pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -43,19 +43,28 @@
   opencascade-override ? null,
 }:
 let
-  wxGTK-prusa = wxGTK32.overrideAttrs (old: {
-    pname = "wxwidgets-prusa3d-patched";
-    version = "3.2.0";
-    configureFlags = old.configureFlags ++ [ "--disable-glcanvasegl" ];
-    patches = [ ./wxWidgets-Makefile.in-fix.patch ];
-    src = fetchFromGitHub {
-      owner = "prusa3d";
-      repo = "wxWidgets";
-      rev = "78aa2dc0ea7ce99dc19adc1140f74c3e2e3f3a26";
-      hash = "sha256-rYvmNmvv48JSKVT4ph9AS+JdstnLSRmcpWz1IdgBzQo=";
-      fetchSubmodules = true;
-    };
-  });
+  wxGTK-prusa =
+    (wxGTK32.override {
+      # prusa-slicer uses an outdated version of wxwidgets that does not support webkitgtk_4_1
+      # While it is possible to cherrypick 1b8664426603376b68f8ca3c54de97ec630e5940 from wxwidgets upstream,
+      # this breaks prusa-slicer cmake in various ways that are complex to fix.
+      #
+      # prusa-slicer does not call webRTC api, which means the conflicting libsoup_2_4 vs libsoup_3 does not cause crashes.
+      webkitgtk_4_1 = webkitgtk_4_0;
+    }).overrideAttrs
+      (old: {
+        pname = "wxwidgets-prusa3d-patched";
+        version = "3.2.0";
+        configureFlags = old.configureFlags ++ [ "--disable-glcanvasegl" ];
+        patches = [ ./wxWidgets-Makefile.in-fix.patch ];
+        src = fetchFromGitHub {
+          owner = "prusa3d";
+          repo = "wxWidgets";
+          rev = "78aa2dc0ea7ce99dc19adc1140f74c3e2e3f3a26";
+          hash = "sha256-rYvmNmvv48JSKVT4ph9AS+JdstnLSRmcpWz1IdgBzQo=";
+          fetchSubmodules = true;
+        };
+      });
```

Comment:
Always sad to see an override like this in Nixpkgs.  Fedora is able to package it with their normal wxGTK-devel package and webkitgtk 4.1.  Can we learn from what they're doing? https://src.fedoraproject.org/rpms/prusa-slicer/blob/rawhide/f/prusa-slicer.spec#_95

Reply:
Actually, hold on, i might be able to drop the prusa-specific wxWidgets completely, using some of the fedora patches. I got past the cmake configure, currently checking which patches are needed to make the whole thing work.

Reply:
Alright, i managed to fix it, pulling in one of the fedora patches.